src/Aquifer.Public.API/Endpoints/Bibles/Texts/Get/Validator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Aquifer.Common.Utilities;
+using Aquifer.Data.Enums;
+using FastEndpoints;
+using FluentValidation;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.Texts.Get;
+
+public class Validator : Validator<Request>
+{
+    public Validator()
```

Comment:
Opinion: I found fluent validation harder to write for some of these more advanced use cases than just standard conditionals at the beginning of my endpoint's handler such as:
```
var validationErrors = new List<string>();
if (myCondition1)
{
    validationErrors.Add("error 1");
}

if (myCondition2)
{
    validationErrors.Add("error 2");
}

if (validationErrors.Any())
    SendErrorResponse(validationErrors);
```

I admittedly had a learning curve.  It seemed like when the rule was easy it was great but when things got complicated it was harder to use than not having the framework.

Reply:
I see the value in at as a first layer of validation. To your point and related to a comment above, if it requires a DB check then it's going to have to be in the endpoint, but the `Validator` is nice since:
1. It's declarative vs imperative.
2. The `Request` that the endpoint gets has been checked at a basic level.

Reply:
These situations are uniquely obnoxious and don't come up too often. Usually the validator is really simple `NotEmpty()` etc. That said, for anything more complex, FastEndpoints has ThrowValidationError or something like that that we can utilize in the endpoint.

Reply:
Thanks for elaborating.  It's helpful to hear the philosophy behind validation to understand why the code is the way it is.